game/src/main/java/com/github/mrpaulblack/tron/MsgError.java

```diff
@@ -14,7 +14,7 @@
 	UNSUPPORTEDPROTOCOLVERSION,
 	SERVERFULL,
 	ILLEGALMOVE,    //brauchen wir diesen Typen überhaupt?
```

Comment:
würde ich rausnehmen. es gibt nur 4 richtungen und die kann man immer ausführen

Reply:
I left it in for 2 reasons: 
1. in case somebody presses a key that is not suppose to be used in the game. 
2. In case somebody presses direction keys while being in spectater mode.

Reply:
Case 2 is valid; Case 1 would not go through the API but gets blocked by the client anyway. I vote to keep the error types tho

Reply:
It's easier to leave it in right now then add it later especially since the tests for it exist now.